app/lib/actions/admins/actions.ts

```diff
@@ -20,6 +20,7 @@ type CompleteAdmin = {
 };
 
 export type AdminState = {
```

Comment:
Tu pourrais faire un CommonState avec isSuccess et message et tu l'ajoutes au ValidateAdminState et AdminState
```suggestion
export type AdminState = {
  errors?: {
    firstName?: string[];
    lastName?: string[];
} & CommonState
```

Reply:
Ok done